harmony-kotlin/src/commonMain/kotlin/com.harmony.kotlin/data/datasource/RetryDataSource.kt

```diff
@@ -0,0 +1,72 @@
+package com.harmony.kotlin.data.datasource
+
+import com.harmony.kotlin.common.logger.Logger
+import com.harmony.kotlin.data.query.Query
+
+class RetryDataSource<V>(
```

Comment:
i've just used the same name it has in swift version:
https://github.com/mobilejazz/harmony-swift/blob/develop/Sources/Harmony/Data/DataSource/RetryDataSource.swift
In any case, I agree that it would be nice to normalize the naming, but we should do23 in both platforms.

Reply:
Ok, let's keep it that way as it already exists in Swift.